core/common/src/ViewProps.ts

```diff
@@ -82,7 +81,6 @@ export interface HydrateViewStateResponseProps {
   sheetViewViews?: (ViewStateProps | undefined)[];
   baseModelProps?: ModelProps;
   spatialViewProps?: ViewStateProps;
-  categoryIdsResult?: SubCategoryResultRow[];
```

Comment:
This is a breaking change.

Reply:
So I guess the right thing to do would be to add it back and mark it as deprecated then? Does that mean I would also have to add back everything involving this? Such as subcategories.preload and postload? And mark those as deprecated as well? I thought things marked with @internal could be freely modified.

Reply:
> I thought things marked with https://github.com/internal could be freely modified.

Not things that are part of Rpc interfaces.